utils/openebs-cleanup.yml

```diff
@@ -34,8 +34,27 @@
   args:
     executable: /bin/bash
 
+- name: Check the openebs components are still there or not
```

Comment:
A better task name would be - `Waiting for graceful termination of openebs pods`